pages/pipelines/migrate_from_jenkins.md

```diff
@@ -0,0 +1,204 @@
+# Migrate from Jenkins
+
+If you are familiar with [Jenkins](https://www.jenkins.io) and want to migrate to Buildkite, this guide is for you. Buildkite is a modern and flexible continuous integration and delivery (CI/CD) platform that provides a powerful and scalable build infrastructure for your applications.
```

Comment:
Curious if we are standardising on "continuous integration and delivery" or "continuous integration and deployment" across our docs?

Reply:
Good catch. I've been looking at this too long so will have missed some things like this. It should be "deployment."

```suggestion
If you are familiar with [Jenkins](https://www.jenkins.io) and want to migrate to Buildkite, this guide is for you. Buildkite is a modern and flexible continuous integration and deployment (CI/CD) platform that provides a powerful and scalable build infrastructure for your applications.
```